### ISSUE TITLE
Add to cart and cart pull-down need to work

### ISSUE THREAD

**Issue by mkobar** [theirfanirfi/angular-ecom#3]
All "add to cart" buttons need to really add the product to the cart (including the detail product/quick-shop dialogs with the -/+ number button).

The cart dialog need to show what the user added to cart and correct totals.  Also the delete buttons on items in the cart needs to work and the totals re-calculated when an item is removed.

The Checkout page should show the real cart data.

The card should also be emptied after a "successful" purchase.

**Comment by theirfanirfi** [comment]
Fixed.

**Comment by mkobar** [comment]
Fixed:

All "add to cart" buttons need to really add the product to the cart

The cart dialog need to show what the user added to cart and correct totals. Also the delete buttons on items in the cart needs to work and the totals re-calculated when an item is removed.

The card should also be emptied after a "successful" purchase. - **this works a little but at times ends up clearing the cart before I purchase or get to the checkout page. Also I have been able to get through the purchase page and the cart was still full.**

Not Fixed:

 (including the **detail product/quick-shop dialogs with the -/+ number button**).

**The Checkout page should show the real cart data.**
